views.md

```diff
@@ -3,14 +3,14 @@
 ## Learning Objectives
 
 - Describe the role of templating languages in a server-side web app
-- Use ERB in sinatra to render views
-- Use ERB tags to include ruby code in HTML templates
+- Use erb in Sinatra to render views
```

Comment:
Angel pro tip: replace first mention of an acronym in LO's with what it stands for. In this case, replace first mention of erb with `"Embedded Ruby" (erb)`